bottles/frontend/views/bottle_details.py

```diff
@@ -102,14 +100,15 @@ def __init__(self, details, config, **kwargs):
         self.window = details.window
         self.manager = details.window.manager
         self.config = config
+        self.check_terminal = False
```

Comment:
```suggestion
```

I put this temporarily because I wanted a quick and dirty way to make it work. This should be a [Try Except](https://www.w3schools.com/python/python_try_except.asp) to only set `self.check_terminal = False` if `self.check_terminal` isn't already set. This Try Except should be under the `run_executable()` function.